src/util.hpp

```diff
@@ -35,6 +35,10 @@ template <typename Int>
 constexpr inline Int cdiv(Int a, int b) { return (a + b - 1) / b; }
 
 #ifdef _WIN32
+#define _HAS_STD_BYTE 0
+#define NOMINMAX
+#include "windows.h"
+#include "processthreadsapi.h"
```

Comment:
if I may be a bit pedantic; `windows.h` and `processthreadsapi.h` should probably be included with `<` and `>`, since they are system headers.
Also, I don't think anything is looking at `_HAS_STD_BYTE`. It has a reserved name, so if anything, I would expect it to be defined by the implementation.

Reply:
fixed